TrkPatRec/src/RobustHelixFinder_module.cc

```diff
@@ -262,6 +264,7 @@ namespace mu2e {
     _useTripletAreaWt(config().UseTripletArea()),
     _chToken     {consumes<ComboHitCollection>(config().ComboHitCollection()) },
     _tcToken     {consumes<TimeClusterCollection>(config().TimeClusterCollection()) },
+    _chfCollTag  {config().chfCollTag()},
```

Comment:
External flag collections are not used consistently between CalPatRec and TrkPatRec.  Fixing that is beyond the scope of this PR, so its OK to add this external collection for now, but it will need to be cleaned up in a future PR.

Reply:
0/0 is undefined.  Best is to use logic to avoid this situation.